service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -47,44 +52,25 @@ private Date convertToDate(OffsetDateTime submissionTimestamp) {
     return null;
   }
 
-  private RunLog convertToRunLog(WorkflowQueryResult queryResult) {
-    // Note: Cromwell's /query endpoint doesn't return 'workflowUrl' or 'workflowInputs' hence
-    // Setting it 'null' for now
+  private RunLog runToRunLog(Run run) {
+
     return new RunLog()
-        .runId(queryResult.getId())
-        .state(convertToRunState(queryResult.getStatus()))
-        .workflowUrl(null)
-        .name(queryResult.getName())
+        .runId(run.id().toString())
+        .workflowUrl(run.runSet().method().methodUrl())
+        .name(run.entityId())
+        .state(convertToRunState(run.status()))
         .workflowParams(null)
```

Comment:
I think this can now be fetched from the database and returned. We might be storing it in `methods` table